wouso/games/teamquest/tests.py

```diff
@@ -203,3 +205,39 @@ def test_quest_status_create(self):
 
     def test_quest_status_time_finished_before_time_started(self):
         pass
+
+
+class TeamQuestViewsTest(WousoTest):
+
+    def setUp(self):
+        self.admin = self._get_superuser()
+        self.c = Client()
+        self.c.login(username='admin', password='admin')
+
+    def test_cpanel_home_view_basic(self):
```

Comment:
It's better if you only test if the Name, Manage and the button exists (just the button string, no glyphicons/trans etc). No need to test for all columns